asm/script_pokemon_util_80A0058.s

```diff
@@ -0,0 +1,535 @@
+	.include "asm/macros.inc"
```

Comment:
Didn't I just delete this file?

Reply:
Beat me to it. Yes, merge error. Fixed now